orc8r/cloud/go/services/configurator/types.go

```diff
@@ -439,6 +439,14 @@ type EntityLoadCriteria struct {
 
 	LoadAssocsToThis   bool
 	LoadAssocsFromThis bool
+
+	// The following parameters allow for pagination of entity loads. These
+	// load criteria parameters should be used in combination with one another.
+
+	// Maximum size of entities returned
+	PageSize uint64
+	// Token for which all entity keys returned are greater than
```

Comment:
This doc comment is peering into implementation details. You can add this as a "for reference this is now it's used", but the leading doc comment should be something like "PageToken is the opaque token provided to load the next page of entities."